tpch/pandas_queries/queries.py

```diff
@@ -549,7 +549,7 @@ def udf(df):
         numerator = df["VOLUME"].sum()
         return numerator / demonimator
 
-    total = total.groupby("O_YEAR").apply(udf).reset_index()
+    total = total.groupby("O_YEAR").apply(udf, include_groups=False).reset_index()
```

Comment:
Why we need `include_groups=False`? [Doc](https://pandas.pydata.org/docs/reference/api/pandas.core.groupby.DataFrameGroupBy.apply.html) says setting include_groups to True is deprecated.

Reply:
The python compiler recommends this, and in the official documentation: Deprecated since version 2.2.0:" Setting include_groups to True is deprecated. Only the value False will be allowed in a future version of pandas."